tests/snmp/test_snmp_lldp.py

```diff
@@ -1,6 +1,11 @@
 import pytest
 from ansible_host import AnsibleHost
 
+@pytest.fixture(scope="module", autouse=True)
+def setup_check_topo(testbed):
+    if testbed['topo']['name'] in ('ptf32','ptf64'):
```

Comment:
this is difficult to maintain. we should at least define a function get_testbed_type(), and only three type should be returned t1, t0, and ptf.

Reply:
@lguohan, ok. I defined get_testbed_type() for it. But I have a question needs your suggestion. I think get_testbed_type() is a common function, right? Where shall I define the new function?

Reply:
@lguohan , got it. I defined get_testbed_type() for it. But I have some questions. What is the type of this function? It's a common function? If yes, where should I define this function? Could you give me some advice? Thanks.

Reply:
Would you please consider adding a field to the object created in `class TestbddInfo` which is defined in tests/conftest.py. For example:

```
class TestbedInfo(object):
    ...

def __init__(self, testbed_file):
    ...
    line['topo']['type'] = self.get_testbed_type()

def get_testbed_type():
    ...
```

Then in test scripts, you can access it using `testbed['topo']['type']`.

Reply:
@wangxin , thank you for your advice. I added a field 'type' to the object created in class TestbddInfo which is defined in tests/conftest.py to identify testbed type. Please review again.